src/main/resources/scripts/pipeline.sh

```diff
@@ -0,0 +1,49 @@
+pipeline {
+    agent {
+        label 'main-node'
+    }
+    
+    tools {
+        jdk 'openjdk-17'
+    }
+    stages {
+        stage('setup') {
+            
+            steps {
+              sh '''
+              wget https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip -O awscliv2.zip
+              unzip -o awscliv2.zip
+              ./aws/install
+              aws configure set aws_access_key_id access_key_id
+              aws configure set aws_secret_access_key access_secret_key
+              aws configure set default.region us-west-2
+              aws configure set default.output json
+              aws ec2 run-instances \
+              --image-id ami-XXXXXXXXXXXXX --count 1 \
```

Comment:
All the values should be read from Jenkins variables like [this](https://github.com/opensearch-project/opensearch-build/blob/main/jenkins/opensearch/perf-test.jenkinsfile#L43).